overrides/android-override.json

```diff
@@ -27,7 +27,20 @@
             "state": "enabled"
         },
         "autofill": {
-            "state": "enabled"
+            "state": "enabled",
+            "settings": {
+                "features": {
+                    "inject_credentials": {
```

Comment:
I believe we use camelcase rather than snake case, @SlayterDev can confirm.

Reply:
Yeah I see that elsewhere now; updated